src/main/java/org/commonjava/util/sidecar/services/ReportService.java

```diff
@@ -31,67 +34,107 @@
 import javax.annotation.PostConstruct;
 import javax.enterprise.context.ApplicationScoped;
 import javax.inject.Inject;
+import javax.ws.rs.core.Response;
+import java.io.File;
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.util.HashMap;
 
 import static org.commonjava.util.sidecar.services.PreSeedConstants.FOLO_BUILD;
+import static org.commonjava.util.sidecar.services.PreSeedConstants.STARTUP_INIT;
 import static org.commonjava.util.sidecar.util.SidecarUtils.getBuildConfigId;
 
-@RegisterForReflection
 @ApplicationScoped
 public class ReportService
 {
     private final Logger logger = LoggerFactory.getLogger( getClass() );
 
-    private TrackedContent trackedContent;
+    public TrackedContent trackedContent;
+
+    private HashMap<String,HistoricalEntryDTO> historicalContentMap;
 
     @Inject
     ObjectMapper objectMapper;
 
+    @Inject
+    Classifier classifier;
+
+    @Inject
+    ProxyService proxyService;
+
     @PostConstruct
     void init(){
         this.trackedContent = new TrackedContent();
+        this.historicalContentMap = new HashMap<>();
     }
 
     public void appendUpload(TrackedContentEntry upload){
         this.trackedContent.appendUpload(upload);
+        logger.info(upload.toString());
```

Comment:
1.Will this cause log overflow if using info level?
2.Makes the log message more meaningful is always helpful.